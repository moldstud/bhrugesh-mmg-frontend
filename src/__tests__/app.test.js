import React from 'react';
import { shallow } from 'enzyme';
import App from '../app.js';

describe('App', () => {
	const wrapper = shallow(<App />);

	it('should render', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
