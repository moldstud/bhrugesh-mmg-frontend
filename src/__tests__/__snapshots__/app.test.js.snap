// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<div>
  <withRouter(Error)>
    <div
      className="row"
    >
      <div
        className="column"
      >
        <Header />
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      >
        <withRouter(Connect(Router)) />
      </div>
    </div>
  </withRouter(Error)>
</div>
`;
